connectathon-scenarios-2019-09/README.md

```diff
@@ -0,0 +1,91 @@
+## Argonaut PAMA Project: September 2019 Connectathon
+
+We'll be participating in the CDS Hooks track of the September 2019
+Connectathon, testing our draft CDS Hooks for PAMA IG.  The track is designed
+for clients and servers that want to try out a full spectrum of integration
+from:
+
+1. Automated rating of orders in-workflow
+2. Suggestions for alternative orders presented in-workflow
+3. App-mediated order selection through SMART Web Messaging
+
+## Scenario 1. Automated rating of orders in-workflow
+
+* EHR provides an order-entry form that triggers the `order-select` hook as
+orders are entered (and optionally triggers `order-sign` when full order
+details are present). Order entry must be able to populate at least: patient,
+provider, and a `draftOrders` bundle containing a `ServiceRequest` with a
```

Comment:
I'd prefer to keep testing as narrowly focused as we can, in part because we need to make sure the sandbox is ready.

Reply:
My comment was primarily around what we're stating as required context during hook invocation.  So, rather than saying "patient", "patientId" would be better so that it's clear we're talking about fields in the hook context.  "selections" is a required field in order-select, but not mentioned here

Reply:
Gotchat, thanks!